assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/BaseAssignmentService.java

```diff
@@ -11835,6 +11835,21 @@ public String getGradeForUserInGradeBook(String userId)
 						Double dblGrade = dcformat.parse(rv).doubleValue();
 						rv = nbFormat.format(dblGrade);
 					}
+					
+					/*
+					 * SAK-32201 - We need to know if 'm_submitterId' is an id of a Group because
```

Comment:
Code doesn't mention {{m_submitterId}} but doco does. What does this actually mean?